bindings_generator/src/class_docs.rs

```diff
@@ -0,0 +1,103 @@
+use std::{collections::HashMap, fs};
+
+use roxmltree::Node;
+
+#[derive(Debug)]
+pub struct GodotXMLDocs {
+    class_fn_desc: HashMap<(String, String), String>,
+}
+
+impl GodotXMLDocs {
+    pub fn new(folder: &str) -> Self {
+        let entries = fs::read_dir(folder)
+            .unwrap()
+            .map(|e| e.unwrap().path())
+            .collect::<Vec<_>>();
+
+        let mut docs = GodotXMLDocs {
+            class_fn_desc: HashMap::default(),
+        };
+
+        for entry in entries {
+            if entry.extension().map(|ext| ext == "xml").is_some() {
+                let file_content =
+                    std::fs::read_to_string(dbg!(entry.as_os_str())).expect("Unable to read file");
```

Comment:
This `dbg` macro invocation should be removed.

Reply:
sorry, done